src/viam/sdk/tests/mocks/mlmodel_mocks.hpp

```diff
@@ -0,0 +1,81 @@
+// Copyright 2023 Viam Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#pragma once
+
+#include <viam/sdk/services/mlmodel/client.hpp>
+#include <viam/sdk/services/mlmodel/mlmodel.hpp>
+#include <viam/sdk/services/mlmodel/server.hpp>
+
+namespace viam {
+namespace sdktests {
+
+class MockMLModelService : public sdk::MLModelService {
+   public:
+    explicit MockMLModelService(std::string name = "mock_mlmodel")
+        : MLModelService(std::move(name)) {}
+
+    std::shared_ptr<named_tensor_views> infer(const named_tensor_views& inputs) override;
+
+    MockMLModelService& metadata(struct metadata metadata);
+    struct metadata metadata() override;
+
+   private:
+    struct metadata metadata_;
+};
+
+#if 0
```

Comment:
I believe this can be removed, it was copypasta from another mock but I think given Zack's new testing approach it isn't needed anymore.

Reply:
This has been removed.